iree_tests/benchmarks/benchmark_sdxl_cpu.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -xeuo pipefail
+
+this_dir="$(cd $(dirname $0) && pwd)"
+iree_root="$(cd $this_dir/.. && pwd)"
+vae_decode_dir="$iree_root/pytorch/models/sdxl-vae-decode-tank"
+scheduled_unet_dir="$iree_root/pytorch/models/sdxl-scheduled-unet-3-tank"
+prompt_encoder_dir="$iree_root/pytorch/models/sdxl-prompt-encoder-tank"
```

Comment:
nit: prefer to wrap variables in `{}` curly braces so they are unambiguous
```suggestion
this_dir="$(cd $(dirname $0) && pwd)"
iree_root="$(cd ${this_dir}/.. && pwd)"
vae_decode_dir="${iree_root}/pytorch/models/sdxl-vae-decode-tank"
scheduled_unet_dir="${iree_root}/pytorch/models/sdxl-scheduled-unet-3-tank"
prompt_encoder_dir="${iree_root}/pytorch/models/sdxl-prompt-encoder-tank"
```

references:
* https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables
* https://google.github.io/styleguide/shellguide.html

---

You can also append a question mark like `${variable?}` to check that the variable is defined instead of defaulting to empty string: https://stackoverflow.com/questions/8889302/what-is-the-meaning-of-a-question-mark-in-bash-variable-parameter-expansion-as-i